styles/globals.css

```diff
@@ -1,9 +1,25 @@
+:root {
+  --primary-color: #141225;
+  --secondary-color: #ffce21;
+  --primary-fontcolor: #fff;
+  --placeholder-color: #8a8a8a;
+  --background-gradient-color: #62606d;
+}
+
 html,
 body {
   padding: 0;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+  font-family: Roboto, sans-serif;
```

Comment:
`Roboto` is not installed on all operating systems by default. Thus, it should be more safe to import that font. Otherwise, `sans-serif` is used.